_gsocproposals/2020/proposal_Phoenix.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Phoenix, interactive data visualization - Development of an experiment independent javascript event display framework and data format
+layout: gsoc_proposal
+project: Phoenix
+year: 2020
+organization: 
+  - UMass
+  - Pittsburgh
+  - CERN
+---
+
+## Description
+
+Visualising HEP event data is currently typically done per experiment (e.g. [VP1](http://atlas-vp1.web.cern.ch/atlas-vp1/home/), [Iguana](https://doi.org/10.1016/j.nima.2004.07.036), [Fireworks](https://iopscience.iop.org/article/10.1088/1742-6596/219/3/032014/pdf)), and normally involves the installation of dedicated software. However modern browsers are more than capable of showing complex detector geometry, as well as representations of the underlying physics. As the [Visualisation](https://arxiv.org/abs/1811.10309) section of HSF Community White Paper explained, using an intermediate data format (e.g. JSON) makes it possible to separate the event display from the underlying (experiment-specific) software framework. 
+
+[Phoenix](https://hepsoftwarefoundation.org/phoenix/) is a framework that can be used by any typical (e.g. colliding beam) High Energy Physics experiment. It was initially based on work done for the TrackML [Kaggle](https://www.kaggle.com/c/trackml-particle-identification)/[Codalab](https://competitions.codalab.org/competitions/20112) challenges (and internal use by ATLAS). We were awarded a GSOC student in 2019 who ported it to [angular](https://angular.io), and add new functionality (such as overlays, compound objects etc). A LHCb example has since been added, so the underlying principle is well validated, and various "loaders" exist to read JSON, Atlas JiveXML, LHCb and TrackML event data. The task now is to make it more user friendly, and to add more functionality.
```

Comment:
"We were awarded" -> "We were working with"

Reply:
"and add new functionality" -> "and added..."

Reply:
Maybe link on "LHCb example"

Reply:
Done! Thanks.